config_hostinit.go

```diff
@@ -46,9 +47,10 @@ func (c *HostInitConfig) Set() error {
 
 	_, err := coll.UpdateOne(ctx, byId(c.SectionId()), bson.M{
 		"$set": bson.M{
-			"ssh_timeout_secs": c.SSHTimeoutSeconds,
-			"host_throttle":    c.HostThrottle,
-			"cloud_batch_size": c.CloudStatusBatchSize,
+			"ssh_timeout_secs":      c.SSHTimeoutSeconds,
```

Comment:
Unrelated to this, but can we make a ticket to delete this because it's not used in host provisioning at all? It hasn't been used in a long time.